inlang/source-code/paraglide/paraglide-js-adapter-unplugin/src/index.ts

```diff
@@ -59,9 +60,14 @@ export const paraglide = createUnplugin((config: UserConfig) => {
 	let project: InlangProject | undefined = undefined
 	async function getProject(): Promise<InlangProject> {
 		if (project) return project
+
+		const repo = await openRepository("file://", {
```

Comment:
@LorisSigrist  this will try to open the whole hard drive as a repo and fail. did you try this or am i missing something?

Reply:
take a look at uses  findRepoRoot eg.    await findRepoRoot({ nodeishFs: fs, path: projectPath })